File: Section8_TF-Library/bumperbot_ws/src/bumperbot_examples/nodes/turtlesim_kinematics_node.cpp
#include "bumperbot_examples/turtlesim_kinematics.h"


int main(int argc, char **argv)
{
    ros::init(argc, argv, "turtlesim_kinematics_node");
    TurtlesimKinematics turtlesim_kinematics;
    ros::spin();

    return 0;
}

File: Section11_Sensor-Fusion/bumperbot_ws/src/bumperbot_examples/nodes/turtlesim_kinematics_node.py
#!/usr/bin/env python3
import rospy
from bumperbot_examples.turtlesim_kinematics import TurtlesimKinematics


if __name__== '__main__':
    rospy.init_node('turtlesim_kinematics_node')
    turtlesim_kinematics = TurtlesimKinematics()

    rospy.spin()